### ISSUE TITLE
Allow custom setup rules

### ISSUE THREAD

**Issue by 130s** [ros-industrial/industrial_ci#14]
Currently in client repo you can add your own script (as documented) like this:
```
:
script: 
  - source .ci_config/travis.sh
  - source ./your_travis.sh
```

`./your_travis.sh` always runs once `.ci_config/travis.sh` finishes. With this you cannot customize the installation setup.

For example, when a device driver package X requires a prorietary library installed to run. This library is available online, but not via `apt` or through package management system and you need to install it in a classic way (unzip, run installer etc.) 
Even if you defined this manual installation in ./your_travis.sh, the library gets installed after `run_tests` and all that already ran.

**Comment by 130s** [comment]
Sorry, posted in the middle by mistake.

----

Currently in client repo you can add your own script (as documented) like this:
```
:
script: 
  - source .ci_config/travis.sh
  - source ./your_travis.sh
```

`./your_travis.sh` always runs once `.ci_config/travis.sh` finishes. With this you cannot customize the installation setup.

For example, when a device driver package X requires a prorietary library installed to run. This library is available online, but not via `apt` or through package management system and you need to install it in a classic way (unzip, run installer etc.) 
Even if you defined this manual installation in ./your_travis.sh, the library gets installed after `run_tests` and all that already ran.

**Comment by 130s** [comment]
Closed via #15
